openpype/tools/workfile_template_build/window.py

```diff
@@ -167,6 +168,18 @@ def _create_option_widgets(self, plugin, options=None):
         self._clear_content_widget()
         attr_defs = plugin.get_placeholder_options(options)
         widget = AttributeDefinitionsWidget(attr_defs, self._content_widget)
+        enum_widgets = widget.findChildren(EnumAttrWidget)
+        for enum_widget in enum_widgets:
+            if enum_widget.attr_def.label == "Family":
+                enum_widget.value_changed.connect(
+                    lambda value: self.on_enum_change(
+                        value,
+                        plugin,
+                        "Family",
+                        "Representation name"
+                    )
+                )
```

Comment:
@iLLiCiTiT what do you think of this logic? Is there a better way?

Reply:
I don't think it's good. It's loader plugin specific logic that won't work with other plugin types.